stratum/hal/lib/phal/onlp/onlpphal.cc

```diff
@@ -131,20 +132,27 @@ ::util::Status OnlpPhal::HandleTransceiverEvent(const TransceiverEvent& event) {
   // made from the upper layer components.
   const std::pair<int, int>& slot_port_pair =
       std::make_pair(event.slot, event.port);
-  auto configurator = slot_port_to_configurator_[slot_port_pair];
 
   // Check to make sure we've got a configurator for this slot/port
-  if (configurator == nullptr) {
+  auto it = slot_port_to_configurator_.find(slot_port_pair);
+  if (it == slot_port_to_configurator_.end()) {
     RETURN_ERROR() << "card[" << event.slot << "]/port[" << event.port << "]: "
                    << "no configurator for this transceiver";
   }
 
-  RETURN_IF_ERROR(configurator->HandleEvent(event.state));
+  RETURN_IF_ERROR(it->second->HandleEvent(event.state));
 
   // Write the event to each writer
   return WriteTransceiverEvent(event);
 }
 
+::util::StatusOr<AttributeDatabaseInterface*> OnlpPhal::GetPhalDB() {
+  if (database_ == nullptr) {
+    RETURN_ERROR() << "database_ is nullptr";
+  }
```

Comment:
This could be simplified to `CHECK_RETURN_IF_FALSE(database_ != nullptr) << "no database";`

Reply:
ok thanks